docs/k8s-quickstart.asciidoc

```diff
@@ -62,11 +62,14 @@ spec:
       node.master: true
       node.data: true
       node.ingest: true
+      node.store.allow_mmap: false
 EOF
 ----
 
 The operator automatically manages Pods and resources corresponding to the desired cluster. It may take up to a few minutes until the cluster is ready.
 
+Note that `node.store.allow_mmap: false` has performance implications and should be tuned for production workloads as described link:k8s-elasticsearch-spec.html#virtual-memory[here].
```

Comment:
Link is broken. I believe this has to be `k8s-virtual-memory.html`

Reply:
That part I wasn't sure about, so was trying to get a doc build locally and failing. Will add some notes in a readme about how to build docs locally once I get it figured out